ha/setup-cluster.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+set -x
+
+WORK_DIR="${1:-$(pwd)}"
+CONFIG_DIR="${2:-"${WORK_DIR}/config"}"
+
+CREATE_VM_SCRIPT="$(pwd)/create-vm.sh"
+
+VM01="node01"
+VM02="node02"
+VM03="node03"
+
+SSH="ssh -o StrictHostKeyChecking=no"
```

Comment:
Let's make this `SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR"` and the same for `sftp` to avoid noise in the logs.